src/components/ImageDetails/RelatedImages.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <aside
+    :aria-label="$t('photo-details.aria.related')"
+    class="p-4 my-6 photo_related-images"
+  >
+    <h3 class="b-header">
+      {{ $t('photo-details.related-images') }}
+    </h3>
+    <ImageGrid
+      :images="images"
+      :can-load-more="false"
+      :is-fetching="$fetchState.pending"
+      :fetching-error="$fetchState.error"
+      :error-message-text="null"
+    />
+  </aside>
+</template>
+
+<script>
+import { ref } from '@nuxtjs/composition-api'
+import useRelated from '~/composables/use-related'
+import { IMAGE } from '~/constants/media'
+
+export default {
+  name: 'RelatedImages',
+  props: {
+    imageId: {
+      type: String,
+      default: null,
+    },
```

Comment:
I would think that at least the `imageId` should be mandatory for this component. What happens if is not present?
```suggestion
    imageId: {
      type: String,
      required: true,
    },
```

Reply:
Definitely true!